auth_session_timeout/models/res_users.py

```diff
@@ -84,7 +84,8 @@ def _auth_timeout_check(self):
             terminated = self._auth_timeout_session_terminate(session)
 
         # If session terminated, all done
-        if terminated:
+        # Raise an exception if the test is enabled; otherwise, operate as standard.
+        if terminated and tools.config["test_enable"]:
```

Comment:
This doesn't look right - why doesn't the Exception need to be raised in normal Odoo operation?